metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -594,15 +594,20 @@
                            (string/join "," cols)
                            seq-name
                            (string/join "," (repeat (count values) "?"))) 
-              ;; replace instances of 'false' with 0 and 'true' with 1 for Postgres 'deleted' column
-              pg-values (map #(if (boolean? %) (if % 1 0) %) values)
+              ;; if set to not use metadata column in Aurora database, then remove it and replace with nil
+              pg-values (if (aurora-config/aurora-metadata?)
+                          values
+                          (map #(if (or (string? %) (number? %) (boolean? %) (nil? %)) % nil) values))
```

Comment:
I guess following on Tyler's question, but why not have a positive check on the data type of the metadata column, rather than a list of "not-metadata-types"?

Reply:
sure, we could do that for readability

Reply:
done!